src/main/resources/application.yml

```diff
@@ -11,4 +11,8 @@ spring:
   flyway:
     enabled: true
     baseline-on-migrate: true
-    locations: classpath:db/migration
\ No newline at end of file
+    locations: classpath:db/migration
+
+  storage:
```

Comment:
This shouldn't be under the "spring" root. Create a separate root (e.g. "auctioneer", "app", ...)